src/script/api/script_text.cpp

```diff
@@ -17,15 +17,10 @@
 
 #include "../../safeguards.h"
 
-RawText::RawText(const char *text) : text(stredup(text))
+RawText::RawText(const char *text) : text(text)
```

Comment:
Can `text` be `nullptr`?

Reply:
Based of my understanding of https://github.com/OpenTTD/OpenTTD/blob/master/cmake/scripts/SquirrelExport.cmake#L38-L47 it cannot be `nullptr`